### PR TITLE
Parse & PayPal configured NEEDS SECRETS ADDED TO WORK PROPERLY

### PR DESCRIPTION
**Pull request by BrainScraps**
You can use card.io, or you can login to your paypal account.

Next I need to handle the response. 

```
    "client": {
    "environment": "sandbox",
    "paypal_sdk_version": "2.9.0",
    "platform": "Android",
    "product_name": "PayPal-Android-SDK"
    },
    "response": {
    "create_time": "2015-04-02T06:21:49Z",
    "id": "PAY-8D299676C6931774HKUON77I",
    "intent": "sale",
    "state": "approved"
    },
    "response_type": "payment"
    }
```

paypalzzz

### CODE REVIEW

File: app/src/main/java/com/zackhsi/kiva/KivaApplication.java [hunk 1]
```diff
@@ -2,18 +2,30 @@
...
+import android.os.Message;
+
+import com.parse.Parse;
+import com.parse.ParseObject;
+import com.zackhsi.kiva.models.PaymentStub;
 
 
 public class KivaApplication extends Application {
     private static Context context;
+    private static String YOUR_APPLICATION_ID = "";
```
Comment (zackhsi) [inline app/src/main/java/com/zackhsi/kiva/KivaApplication.java:15]: There's an `oauth-part-2` branch that has this file fleshed out

**Comment by zackhsi** [comment]
Solid. @danielhochman and I will crank on OAuth a bit today, then hopefully all the pieces will come together

**Comment by zackhsi** [comment]
:+1:

**Comment by danielhochman** [comment]
:open_mouth:

**Pull request merged**